include/circt/Dialect/FIRRTL/FIRRTLDialect.h

```diff
@@ -22,9 +22,11 @@ namespace firrtl {
 
 class FIRRTLType;
 
-/// If the specified attribute list has a firrtl.name attribute, return its
-/// value.
-StringAttr getFIRRTLNameAttr(ArrayRef<NamedAttribute> attrs);
+const StringRef portNameAttrStringRef("portNames");
```

Comment:
We don't typically do this for other attributes.  Please just use the magic constant where necessary.  Thanks.